src/cmd/manage_nsfs.js

```diff
@@ -754,6 +765,50 @@ async function validate_account_args(data, action) {
     }
 }
 
+// we take the data object and extract its properties according to the schema:
+// - we unwrap ths sensitive string (that we have from fetch_account_data)
+// - we change the creation_date to string
+function extract_properties_from_account_data_and_validate_schema(data) {
+    try {
+        const account_to_validate = {
```

Comment:
this is redundant, JSON.stringify will do it for you

Reply:
Thanks, would change it.